components/NavigationBar.js

```diff
@@ -22,8 +24,37 @@ export default function NavigationBar () {
 function NavigationLink ({ href, disabled, children }) {
   return (
     <span className={disabled ? 'disabled' : ''}>
-      <LevitatingLink href={href}>{children}</LevitatingLink>
-      <style jsx>{`
+      <Link to={href} spy smooth duration={500}>
+          <div className='nav-link'>{children}</div>
```

Comment:
`<div>` should not be used inside `<span>` because it violates the nesting rules in HTML specification.